docs/translations/cn/gaia/delegator-guide-cli.md

```diff
@@ -399,20 +399,20 @@ fees = gas * gasPrices
 
 ```bash
 // 向指定验证人绑定一定数量的Atom通证
-// 参数设定样例: <validatorAddress>=cosmosvaloper18thamkhnj9wz8pa4nhnp9rldprgant57pk2m8s, <amountToBound>=10000stake, <gasPrice>=0.001stake
+// 参数设定样例: <validatorAddress>=cosmosvaloper18thamkhnj9wz8pa4nhnp9rldprgant57pk2m8s, <amountToBound>=10000atom, <gasPrice>=0.001atom
```

Comment:
all changes in this document should use `uatom` instead of `atom`

Reply:
Should I scale up the amount such as  `<amountToBound>=10000000000uatom, <gasPrice>=1000uatom` ?